.gitignore

```diff
@@ -2,3 +2,5 @@
 .idea
 
 node_modules/*
+
+composer.lock
```

Comment:
Lock files should not be excluded from sources, while vendor files should be excluded.